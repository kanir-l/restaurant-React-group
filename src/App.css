.footer-container {
  width: 100%;
  background-color: #6e6666;
  height: 10vh;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
}

.footer-container .social-media {
  margin: 10px;
}

.header-container {
  width: 100%;
  height: 10vh;
  background-color: white;
  background-position: center;
  background-size: cover;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
  -webkit-box-pack: justify;
      -ms-flex-pack: justify;
          justify-content: space-between;
}

.header-container nav {
  width: 600px;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
}

.admin-container {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
      -ms-flex-pack: center;
          justify-content: center;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
      -ms-flex-direction: column;
          flex-direction: column;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
}

.booking-container {
  width: 90%;
}

.booking-box {
  margin-top: 24px;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-align: center;
      -ms-flex-align: center;
          align-items: center;
}

.booking {
  width: 600px;
}

.edit-delete {
  width: 75px;
}

b,
p {
  display: inline;
  width: 90%;
}

html,
body {
  margin: 0;
  padding: 0;
}

*,
*::after,
*::before {
  -webkit-box-sizing: border-box;
          box-sizing: border-box;
}

body {
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
  height: 100vh;
  text-rendering: optimizeLegibility;
}

.logo,
li,
a {
  display: inline-block;
  text-decoration: none;
  color: #cacaca;
  margin: 10px;
}

/* Mobile Device */
/* Admin */
/*# sourceMappingURL=App.css.map */