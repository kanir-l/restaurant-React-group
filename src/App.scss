//// Components ////
@import "./scss/components/footer";
@import "./scss/components/header";
//// Pages //////
@import "./scss/pages/admin";
@import "./scss/pages/confirmation";
@import "./scss/pages/contact";
@import "./scss/pages/reservation";
@import "./scss/pages/start";
//// Utilities ////
@import "./scss/utilities/base";
@import "./scss/utilities/mixins";
@import "./scss/utilities/variables";