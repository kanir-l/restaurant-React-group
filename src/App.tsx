import React from 'react';
import './App.scss';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
import Footer from './components/Footer';
import Header from './components/Header';
import Start from './pages/Start';
import Reservations from './pages/Reservations';
import Contact from './pages/Contact';
import Admin from './pages/Admin';
import Confirmation from './pages/Confirmation';
import Booking from './pages/Booking';
import { faFacebook, faInstagram, faTwitter } from '@fortawesome/free-brands-svg-icons';
import { library } from '@fortawesome/fontawesome-svg-core';
import Cancellation from './pages/Cancellation';

library.add(faFacebook, faInstagram, faTwitter);

function App() {
  return (
    <div className="App">

      <Router>
        <div className="container">
        <Header></Header>

        <Switch>
          <Route path='/' exact>
            <Start></Start>
          </Route>
        </Switch>

        <Switch>
          <Route path='/reservations' exact>
            <Reservations></Reservations>
          </Route>
        </Switch>

        <Switch>
          <Route path='/reservations/confirmation'>
            <Confirmation></Confirmation>
          </Route>
        </Switch>

        <Switch>
          <Route path='/reservations/delete/:id' exact>
            <Cancellation></Cancellation>
          </Route>
        </Switch>

        <Switch>
          <Route path='/contact' exact>
            <Contact></Contact>
          </Route>
        </Switch>

        <Switch>
          <Route path='/admin' exact>
            <Admin></Admin>
          </Route>
        </Switch>

        <Switch>
          <Route path='/admin/:id' exact>
            <Booking></Booking>
          </Route>
        </Switch>

        <Footer></Footer>
        </div>
      </Router>
    </div>
  );
}

export default App;
