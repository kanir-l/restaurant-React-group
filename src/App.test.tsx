 import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';
import { Router } from 'react-router';
import {createMemoryHistory} from 'history'

// Checking Navigation in the header
test('start should go the the correct route', () => {
  const history = createMemoryHistory()
  history.push('/')
  render(
    <Router history={history}>
      <App />
    </Router>,
  )

  expect(screen.getByText(/L'Isola/i)).toBeInTheDocument()
})  

test('reservations should go the the correct route', () => {
  const history = createMemoryHistory()
  history.push('/reservations')
  render(
    <Router history={history}>
      <App />
    </Router>,
  )

  expect(screen.getByText(/reservations/i)).toBeInTheDocument()
}) 

test('admin should go the the correct route', () => {
  const history = createMemoryHistory()
  history.push('/contact')
  render(
    <Router history={history}>
      <App />
    </Router>,
  )

  expect(screen.getByText(/contact/i)).toBeInTheDocument()
})  
