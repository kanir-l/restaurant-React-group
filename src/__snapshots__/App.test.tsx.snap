// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reservations renders correctly 1`] = `
<div
  className="reservations-container"
>
  <img
    alt="Plate of mussels and soup"
    className="reservations-img"
    src="images/reservations-photo.jpg"
  />
  <div
    className="reservations"
  >
    <div>
      <div
        className="input-guests-container"
      >
        <h4>
          Number of guests
        </h4>
        <button
          disabled={true}
          onClick={[Function]}
        >
           - 
        </button>
        <input
          id="numberOfGuests"
          max="90"
          min="1"
          name="numberOfGuests"
          onChange={[Function]}
          type="number"
          value="1"
        />
        <button
          onClick={[Function]}
        >
           + 
        </button>
      </div>
      <div
        className="input-date-container"
      >
        <h4>
          Date
        </h4>
        <div
          className="react-calendar"
        >
          <div
            className="react-calendar__navigation"
            style={
              Object {
                "display": "flex",
              }
            }
          >
            <button
              aria-label=""
              className="react-calendar__navigation__arrow react-calendar__navigation__prev-button"
              disabled={true}
              onClick={[Function]}
              type="button"
            >
              ‹
            </button>
            <button
              aria-label=""
              className="react-calendar__navigation__label"
              disabled={false}
              onClick={[Function]}
              style={
                Object {
                  "flexGrow": 1,
                }
              }
              type="button"
            >
              <span
                className="react-calendar__navigation__label__labelText react-calendar__navigation__label__labelText--from"
              >
                September 2021
              </span>
            </button>
            <button
              aria-label=""
              className="react-calendar__navigation__arrow react-calendar__navigation__next-button"
              disabled={false}
              onClick={[Function]}
              type="button"
            >
              ›
            </button>
          </div>
          <div
            className="react-calendar__viewContainer"
            onBlur={null}
            onMouseLeave={null}
          >
            <div
              className="react-calendar__month-view"
            >
              <div
                style={
                  Object {
                    "alignItems": "flex-end",
                    "display": "flex",
                  }
                }
              >
                <div
                  style={
                    Object {
                      "flexGrow": 1,
                      "width": "100%",
                    }
                  }
                >
                  <div
                    className="react-calendar__month-view__weekdays"
                    onFocus={null}
                    onMouseOver={null}
                    style={
                      Object {
                        "display": "flex",
                        "flexDirection": undefined,
                        "flexWrap": "no-wrap",
                      }
                    }
                  >
                    <div
                      className="react-calendar__month-view__weekdays__weekday"
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                    >
                      <abbr
                        aria-label="Monday"
                        title="Monday"
                      >
                        Mon
                      </abbr>
                    </div>
                    <div
                      className="react-calendar__month-view__weekdays__weekday"
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                    >
                      <abbr
                        aria-label="Tuesday"
                        title="Tuesday"
                      >
                        Tue
                      </abbr>
                    </div>
                    <div
                      className="react-calendar__month-view__weekdays__weekday"
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                    >
                      <abbr
                        aria-label="Wednesday"
                        title="Wednesday"
                      >
                        Wed
                      </abbr>
                    </div>
                    <div
                      className="react-calendar__month-view__weekdays__weekday"
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                    >
                      <abbr
                        aria-label="Thursday"
                        title="Thursday"
                      >
                        Thu
                      </abbr>
                    </div>
                    <div
                      className="react-calendar__month-view__weekdays__weekday"
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                    >
                      <abbr
                        aria-label="Friday"
                        title="Friday"
                      >
                        Fri
                      </abbr>
                    </div>
                    <div
                      className="react-calendar__month-view__weekdays__weekday"
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                    >
                      <abbr
                        aria-label="Saturday"
                        title="Saturday"
                      >
                        Sat
                      </abbr>
                    </div>
                    <div
                      className="react-calendar__month-view__weekdays__weekday"
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                    >
                      <abbr
                        aria-label="Sunday"
                        title="Sunday"
                      >
                        Sun
                      </abbr>
                    </div>
                  </div>
                  <div
                    className="react-calendar__month-view__days"
                    style={
                      Object {
                        "display": "flex",
                        "flexDirection": undefined,
                        "flexWrap": "wrap",
                      }
                    }
                  >
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--neighboringMonth"
                      disabled={true}
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="August 30, 2021"
                      >
                        30
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--neighboringMonth"
                      disabled={true}
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="August 31, 2021"
                      >
                        31
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      disabled={true}
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 1, 2021"
                      >
                        1
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      disabled={true}
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 2, 2021"
                      >
                        2
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      disabled={true}
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 3, 2021"
                      >
                        3
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                      disabled={true}
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 4, 2021"
                      >
                        4
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                      disabled={true}
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 5, 2021"
                      >
                        5
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      disabled={true}
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 6, 2021"
                      >
                        6
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      disabled={true}
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 7, 2021"
                      >
                        7
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      disabled={true}
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 8, 2021"
                      >
                        8
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__tile--now react-calendar__tile--active react-calendar__tile--range react-calendar__tile--rangeStart react-calendar__tile--rangeEnd react-calendar__tile--rangeBothEnds react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 9, 2021"
                      >
                        9
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 10, 2021"
                      >
                        10
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 11, 2021"
                      >
                        11
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 12, 2021"
                      >
                        12
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 13, 2021"
                      >
                        13
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 14, 2021"
                      >
                        14
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 15, 2021"
                      >
                        15
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 16, 2021"
                      >
                        16
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 17, 2021"
                      >
                        17
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 18, 2021"
                      >
                        18
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 19, 2021"
                      >
                        19
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 20, 2021"
                      >
                        20
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 21, 2021"
                      >
                        21
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 22, 2021"
                      >
                        22
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 23, 2021"
                      >
                        23
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 24, 2021"
                      >
                        24
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 25, 2021"
                      >
                        25
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 26, 2021"
                      >
                        26
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 27, 2021"
                      >
                        27
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 28, 2021"
                      >
                        28
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 29, 2021"
                      >
                        29
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="September 30, 2021"
                      >
                        30
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--neighboringMonth"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="October 1, 2021"
                      >
                        1
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend react-calendar__month-view__days__day--neighboringMonth"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="October 2, 2021"
                      >
                        2
                      </abbr>
                    </button>
                    <button
                      className="react-calendar__tile react-calendar__month-view__days__day react-calendar__month-view__days__day--weekend react-calendar__month-view__days__day--neighboringMonth"
                      onClick={[Function]}
                      onFocus={null}
                      onMouseOver={null}
                      style={
                        Object {
                          "flexBasis": "14.285714285714286%",
                          "marginLeft": null,
                          "maxWidth": "14.285714285714286%",
                          "overflow": "hidden",
                        }
                      }
                      type="button"
                    >
                      <abbr
                        aria-label="October 3, 2021"
                      >
                        3
                      </abbr>
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <button
        className="main-btn-disabled"
        disabled={true}
      >
        Continue
      </button>
    </div>
  </div>
</div>
`;
